Framework/Algorithms/test/GeneralisedSecondDifferenceTest.h

```diff
@@ -33,8 +36,8 @@ class GeneralisedSecondDifferenceTest : public CxxTest::TestSuite {
                              10.3, 3.9, 0.47, 0.3,  0.3};
     MatrixWorkspace_sptr inputWs = WorkspaceFactory::Instance().create(
         "Workspace2D", 1, y.size(), y.size());
-    inputWs->dataY(0) = y;
-    inputWs->dataX(0) = x;
+    inputWs->mutableY(0) = y;
```

Comment:
👍, but one note:
If you use `setHistogram` *without* specifying standard deviations or variances as third argument the will be *auto-generated based on the Y values*. In this case this will be different  from the default values used in this test. I would just see if the test still passes, it might be insensitive to changes in the error bars.